text/0000-standardize-error-docs.md

```diff
@@ -0,0 +1,55 @@
+- Feature Name: standardize_error_docs
+- Start Date: 2023-01-10
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+
+# Summary
+[summary]: #summary
+
+(Re)-standardize long-form error code explanations (referred to in this RFC as "error docs") with a more flexible system that also takes into account new errors which do not fit into the format as defined by [RFC 1567]. This RFC replaces [RFC 1567] as the primary specification for error docs. This RFC ultimately wishes to make error docs more accessible and easy to read for both new and proficient Rust users.
```

Comment:
You wrote that "This RFC replaces [RFC 1567]", however it doesn't cover it at all. I'd say it "extends" it instead (or any similar word to say it only replaces the outdated parts).

Reply:
Ah hold on, just read below that you want a style spec instead of an RFC for the style, my bad.

Reply:
RFC 1567 just specifies a style for the error docs. This RFC does that? Isn't all of RFC 1567 outdated in terms of this RFC?